inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,108 @@
+#Pseudocode may look like actual code...please ignore...
+get_pdb_ids <- function(target_seq, start_pos = 1 , end_pos = nchar(target_seq), convert = TRUE, silent = TRUE){
+  
+  #get packages
+  # install.packages("bio3d")
+  # install.packages("RCurl")
+  # install.packages("XML")
+  # install.packages("seqinr")
+  # library(bio3d)
+  # library(RCurl)
+  # library(XML)
+  # library(seqinr)
+  
+  
+  
+  #convert sequence into AA 
+  if(convert){
+    #translate needs a vector of single chars, not a string
+    AA_seq <- paste(translate(s2c(target_seq)), collapse='')
+    end_pos <- nchar(AA_seq)
+  }else{ AA_seq <- target_seq }
+ 
+  blast_output <- blast.pdb(AA_seq)
+  
+  blast_output <- blast_output$hit.tbl
+  blast_output <- blast_output[blast_output$identity > 40, ]
+  
+  if(nrow(blast_output) == 0){
+    return(NULL)
+  }
+  
+  blast_output$tax <- NA 
+  blast_output$info <- NA
+  
+  final_output <- data.frame()
+  final_output_range <- c()
+  final_output_range_counter <- 1
+  
+  for(entry in 1:nrow(blast_output)){
+    rcsb_id <- gsub("_", ".", blast_output$pdb.id[entry], ignore.case = TRUE)
```

Comment:
See this is precisely where pseudocode would have been preferred: I happen to know what you are doing here, but for everyone else this is completely unnecessarily opaque. What's keeping you from writing?
```
# Reformat PDB ID as output by BLAST to separate ID from chain with a "."
```
This has nothing to do with getting "carried away" with writing code - and everything to do with not being explicit in this conversation. 

(Incidentally: why is this even necessary?)